content.md

```diff
@@ -1,5 +1,40 @@
 # Testing your code with `MiniTest`
-Testing is a critical skill in software development. It helps ensure your code works as expected and remains reliable as it evolves. In this lesson, we'll introduce [`MiniTest`](https://github.com/minitest/minitest), a testing framework incldued with Ruby, to write our first unit tests.
+Testing is a critical skill in software development. It helps ensure your code works as expected and remains reliable as it evolves. In this lesson, we'll start by discussing the fundamentals of software testing, including how to gather requirements, write functional requirements using user stories, and develop test plans. Then, we'll introduce [`MiniTest`](https://github.com/minitest/minitest), a testing framework included with Ruby, to write our first unit tests.
```

Comment:
```suggestion
Testing is a critical skill in software development. It helps ensure your code works as expected and remains reliable as it evolves. In this lesson, we'll start by discussing the fundamentals of software testing, including how to gather requirements, write functional requirements using user stories, and develop test plans. Then, we'll introduce the [`MiniTest`](https://github.com/minitest/minitest) gem to write our first unit tests.
```

Even if it is true that it's included with Ruby, I think this aligns better with the "create a Gemfile" step, and reinforces the concept of "gems"

Reply:
I realize it is true that it's included with Ruby, so we're definitely lying a bit here with the Gemfile section, but it just fits so nicely with the concept of gems that was just introduced a few lessons ago. What do you think?

Reply:
I'm ok telling a little fib to teach a point